quinn-proto/src/packet.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
 // to inspect the version and packet type (which depends on the version).
 // This information allows us to fully decode and decrypt the packet.
 #[allow(unreachable_pub)] // fuzzing only
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Where/why do we need these `Clone` derives?

Reply:
These are necessary for the tests, in this line: https://github.com/quinn-rs/quinn/blob/8244dedb0b5b1fda8ca3f9869fd299b6aac6cfdc/quinn-proto/src/connection/mod.rs#L3313

Reply:
Ah... How do you feel about doing a `cfg_attr()` thing here?

Reply:
Sounds reasonable, will add it